surround/cli.py

```diff
@@ -456,9 +459,11 @@ def execute_cli():
     remote_cli.add_push_parser(sub_parser)
     remote_cli.add_list_parser(sub_parser)
 
+    visualise_parser = sub_parser.add_parser('visualise', parents=[visualise_cli.get_visualise_parser()], add_help=False, help="Visualise results of a pipeline")
+
     # Check for valid sub commands as 'add_subparsers' in Python < 3.7
     # is missing the 'required' keyword
-    tools = ["init", "lint", "run", "remote", "add", "pull", "push", "list"]
+    tools = ["init", "lint", "run", "remote", "add", "pull", "push", "list", "visualise"]
```

Comment:
`visualise` is too long, what about `viz`?

Reply:
Yeah sure 👍